docs/trulens_eval/api/provider/openai/azureopenai.md

```diff
@@ -1,10 +1,12 @@
 # AzureOpenAI Provider
 
-Below is how you can instantiate AzureOpenAI as a provider.
+Below is how you can instantiate _AzureOpenAI_ as a provider.
```

Comment:
is AzureOpenAI a proper name or just reference to a class?

Reply:
Proper name would be _Azure OpenAI_ (with space). Perhaps here I'll add the space to reference the proper name rather then the class?

Reply:
Yes, and if it is a class, you can use the reference [AzureOpenAi][...full path to class] instead.